cmd/amp/main.go

```diff
@@ -85,10 +96,11 @@ func main() {
 	RootCmd.SetUsageTemplate(usageTemplate)
 	RootCmd.SetHelpTemplate(helpTemplate)
 
-	RootCmd.PersistentFlags().StringVar(&configFile, "config", "", "Config file (default is $HOME/.config/amp/amp.yaml)")
+	RootCmd.PersistentFlags().StringVar(&configFile, "config", "$HOME/.config/amp/amp.yaml", "Config file")
```

Comment:
Problem is, it breaks the use case when the user uses an env variable for XDG_CONFIG_HOME (not sure who would do that), not a big deal but that means that we have some dead code.